### ISSUE TITLE
📁Docs: Component `Tags-Input` is missing the accessibility documentation for `Enter`

### ISSUE THREAD

**Issue by mmailaender** [melt-ui/melt-ui#876]
**Describe the bug**
Component `Tags-Input` is missing the accessibility documentation for `Enter`
![image](https://github.com/melt-ui/melt-ui/assets/87228994/302a949a-ed72-4b94-884e-5c154698e225)

![image](https://github.com/melt-ui/melt-ui/assets/87228994/447b6690-df56-45f9-8aff-4d060ba6ddde)
`Enter` allows to change the name of an existing tag
